module/VuFind/src/VuFind/Search/SolrAlternative/Results.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+/**
+ * Search Results for second Solr index
+ *
+ * PHP version 7
+ *
+ * Copyright (C) Staats- und Universitätsbibliothek Hamburg 2018.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ *
+ * @category VuFind
+ * @package  Search_SolrAlternative
+ * @author   Hajo Seng <hajo.seng@sub.uni-hamburg.de>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org Main Page
+ */
+namespace VuFind\Search\SolrAlternative;
+
+/**
+ * Search Results for second Solr index
+ *
+ * @category VuFind
+ * @package  Search_SolrAlternative
+ * @author   Hajo Seng <hajo.seng@sub.uni-hamburg.de>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org Main Site
+ */
+class Results extends \VuFind\Search\Solr\Results
+{
+    /**
+     * Support method for performAndProcessSearch -- perform a search based on the
+     * parameters passed to the object.
+     *
+     * @return void
+     */
+    protected function performSearch()
+    {
+        $query  = $this->getParams()->getQuery();
+        $limit  = $this->getParams()->getLimit();
+        $offset = $this->getStartRecord() - 1;
+        $params = $this->getParams()->getBackendParameters();
+        $collection = $this->getSearchService()->search(
+            'SolrAlternative', $query, $offset, $limit, $params
```

Comment:
Is there a particular reason that you overrode this function? It seems like the base version in \VuFind\Search\Solr\Results uses $this->backendId and thus should be able to communicate with 'SolrAlternative' without the value being hard-coded here. Apart from that change, you seem to have removed some functionality, like spelling support, that might be useful. Was that necessary? Is there a better solution?

Reply:
I have to have a closer look at this issue